### ISSUE TITLE
Add sub navigation component to ComponentPage

### ISSUE THREAD

**Issue by nikek** [spotify/backstage#1040]
Make use of the sub navigation component (#922) and add it to the ComponentPage with links that go nowhere. We'll solve routing in a separate issue.

Depends on #922 

![image](https://user-images.githubusercontent.com/3097461/82799519-74bd8e80-9e7a-11ea-9f52-4c33d925fce9.png)
